commerce-add-ons/payment-providers/invoicing/README.md

```diff
@@ -9,11 +9,11 @@ The Invoicing payment provider is a "pass-through" payment provider that doesn't
 ## [Install Invoice & Zero Payment Providers](../install-payment-providers.md)
 
 {% hint style="info" %}
-The invoice and zero payment methods is installed with Umbraco Commerce by default. Follow the installation instruction if for some reason the package has been removed from your implementation.
+By default, Umbraco Commerce includes the Invoice and Zero payment methods. If the package has been removed from your implementation, see the [Installing Umbraco Commerce](https://docs.umbraco.com/umbraco-commerce/installation/install) article to reinstall it again.
```

Comment:
If the payment method is included with Commerce by default, does it make sense to link to installation docs (see line 9)?

Reply:
Good question. I have removed the link from line 9 as it's relevant for other payment providers not the default one's 🙂